### PR TITLE
Add 'conan' package recipe

### PR DESCRIPTION
**Pull request by renatofilho**

### CODE REVIEW

File: conan/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    version = "1.2"
+    license = "https://raw.githubusercontent.com/KDAB/KDStateMachineEditor/1.2/LICENSE.txt"
+    author = "Renato Araujo Oliveira Filho <renato.araujo@kdab.com>"
+    url = "https://github.com/KDAB/KDStateMachineEditor.git"
+    description = "The KDAB State Machine Editor Library is a framework that can be used to help develop full-featured State Machine Editing graphical user interfaces and tools."
+    generators = "cmake"
+
+    def requirements(self):
+        self.requires("qt/5.13.2@kdab/stable")
+        self.requires("KDChart/2.6.50@kdab/stable")
```
Comment (winterz) [inline conan/conanfile.py]: does not use kdchart

File: conan/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+##  clear to you.
+###############################################################################
+
+from conans import ConanFile, CMake, tools
+
+
+class KDStateMachineEditorConan(ConanFile):
+    name = "KDStateMachineEditor"
+    version = "1.2"
+    license = "https://raw.githubusercontent.com/KDAB/KDStateMachineEditor/1.2/LICENSE.txt"
```
Comment (winterz) [inline conan/conanfile.py]: LICENSE.txt is the commercial license.  This is a dual-licensed product and can have either a commercial license (LICENSE.txt) or an LGPL license (LICENSE.LGPL.txt).  Can we have both licenses mentioned somehow?

File: conan/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    author = "Renato Araujo Oliveira Filho <renato.araujo@kdab.com>"
```
Comment (winterz) [inline conan/conanfile.py]: change to  Klaralvdalens Datakonsult AB (KDAB) <info@kdab.com>

File: conan/conanfile.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        self.options["qt"].qtsvg = True
+        self.options["qt"].qtdeclarative = True
+        self.options["qt"].qtremoteobjects = True
+        self.options["qt"].qtscxml = True
+
+    def build(self):
+        self.cmake = CMake(self)
+        self.cmake.definitions["WITH_INTERNAL_GRAPHVIZ"] = True
+        self.cmake.definitions["BUILD_EXAMPLES"] = False
+        self.cmake.definitions["BUILD_DOCS"] = False
```
Comment (winterz) [inline conan/conanfile.py]: can we pass the cmake options to the conan install command for installing the docs or examples?
someone may want to build the docs or examples?

Also I see that we should have a cmake option to disable building the tests. I 'll add that to the kdstatemachineeditor/CMakeLists.txt

Comment (winterz) [reply conan/conanfile.py]: I just added a new cmake option 'BUILD_TESTS' to the 1.2 and master branches.
Please add support for BUILD_TESTS.

**Comment by renatofilho** [comment]
All comments fixed.

**Pull request merged**